packages/cli/generators/discover/index.js

```diff
@@ -180,6 +191,66 @@ module.exports = class DiscoveryGenerator extends ArtifactGenerator {
     });
   }
 
+  /**
+   * Prompts what naming convention they would like to have for column names.
+   */
+  promptColNamingConvention() {
+    this.namingConvention = [
+      'Camel case (exampleColumn) (Recommended)',
+      'No conversion (EXAMPLE_COLUMN)',
+    ];
```

Comment:
Nitpick, it would be even better to have:
```ts
this.namingConvention = [
      {
        name: 'Camel case (exampleColumn) (Recommended)', 
        value: 'camelCase',
      },
      {
        name: 'No conversion (EXAMPLE_COLUMN)', 
        value: 'noCase',
      },
    ];
```